Doc/library/subprocess.rst

```diff
@@ -839,18 +851,87 @@ The :mod:`subprocess` module exposes the following constants.
    additional information.
 
 .. data:: CREATE_NEW_CONSOLE
-
+  
    The new process has a new console, instead of inheriting its parent's
    console (the default).
 
 .. data:: CREATE_NEW_PROCESS_GROUP
-
+  
+   .. versionadded:: 3.7
    A :class:`Popen` ``creationflags`` parameter to specify that a new process
    group will be created. This flag is necessary for using :func:`os.kill`
    on the subprocess.
 
    This flag is ignored if :data:`CREATE_NEW_CONSOLE` is specified.
-
+  
+.. data:: ABOVE_NORMAL_PRIORITY_CLASS
+  
+   .. versionadded:: 3.7
```

Comment:
The versionadded markup is usually added after the text, and surrounded by an empty line (above and after).